tests/utils_tests/test_safestring.py

```diff
@@ -24,6 +24,13 @@ def test_mark_safe(self):
         self.assertRenderEqual('{{ s }}', 'a&b', s=s)
         self.assertRenderEqual('{{ s|force_escape }}', 'a&amp;b', s=s)
 
+    def test_mark_safe_str(self):
+        """
+        Calling str() on a SafeText instance doesn't lose the safe status.
+        """
+        s = mark_safe('a&b')
+        self.assertIsInstance(str(s), type(s))
```

Comment:
I think the test is more readable with `type(s)` -> `SafeData`, but I'll leave the choice to you.